[2022-11-04T19:14:45.873+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: orquestacion.TASK1 scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-11-04T19:14:45.909+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: orquestacion.TASK1 scheduled__2022-05-27T00:00:00+00:00 [queued]>
[2022-11-04T19:14:45.910+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-04T19:14:45.912+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-04T19:14:45.915+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-04T19:14:45.959+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): TASK1> on 2022-05-27 00:00:00+00:00
[2022-11-04T19:14:45.977+0000] {standard_task_runner.py:55} INFO - Started process 4783 to run task
[2022-11-04T19:14:46.010+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'orquestacion', 'TASK1', 'scheduled__2022-05-27T00:00:00+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/5.1-orquestacion.py', '--cfg-path', '/tmp/tmpxgf7ed54']
[2022-11-04T19:14:46.012+0000] {standard_task_runner.py:83} INFO - Job 56: Subtask TASK1
[2022-11-04T19:14:46.260+0000] {task_command.py:376} INFO - Running <TaskInstance: orquestacion.TASK1 scheduled__2022-05-27T00:00:00+00:00 [running]> on host c7d12af55512
[2022-11-04T19:14:46.455+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=orquestacion
AIRFLOW_CTX_TASK_ID=TASK1
AIRFLOW_CTX_EXECUTION_DATE=2022-05-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-27T00:00:00+00:00
[2022-11-04T19:14:46.461+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-04T19:14:46.464+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "SLEEP 2 && ECHO 'TASK1'"]
[2022-11-04T19:14:46.502+0000] {subprocess.py:86} INFO - Output:
[2022-11-04T19:14:46.539+0000] {subprocess.py:93} INFO - /bin/bash: line 1: SLEEP: command not found
[2022-11-04T19:14:46.545+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-11-04T19:14:46.578+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-11-04T19:14:46.597+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=orquestacion, task_id=TASK1, execution_date=20220527T000000, start_date=20221104T191445, end_date=20221104T191446
[2022-11-04T19:14:46.656+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 56 for task TASK1 (Bash command failed. The command returned a non-zero exit code 127.; 4783)
[2022-11-04T19:14:46.703+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-11-04T19:14:47.078+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
